clients/src/test/java/org/apache/kafka/common/utils/UtilsTest.java

```diff
@@ -55,6 +57,21 @@
 
 public class UtilsTest {
 
+    @Test
+    public void testMurmur2() {
+        Map<byte[],Integer> cases = new java.util.HashMap<>();
```

Comment:
Should we import it explicitly?

Reply:
I can do that... Is this the reason the `clients:checkstyleTest` fails? I can't view the file to the report listing the exact violation.

Reply:
```
13:36:48 FAILURE: Build failed with an exception.
13:36:48 
13:36:48 * What went wrong:
13:36:48 Execution failed for task ':clients:checkstyleTest'.
13:36:48 > Checkstyle rule violations were found. See the report at: file:///home/jenkins/jenkins-slave/workspace/kafka-pr-jdk8-scala2.11/clients/build/reports/checkstyle/test.html
13:36:48   Checkstyle files with violations: 1
13:36:48   Checkstyle violations by severity: [error:5]
```

Reply:
There are a few checkstyle failures:
```
[ant:checkstyle] [ERROR] .../UtilsTest.java:62:20: ',' is not followed by whitespace. [WhitespaceAfter]
[ant:checkstyle] [ERROR] .../UtilsTest.java:68:34: ',' is not followed by whitespace. [WhitespaceAfter]
[ant:checkstyle] [ERROR] .../UtilsTest.java:68:38: ',' is not followed by whitespace. [WhitespaceAfter]
[ant:checkstyle] [ERROR] .../UtilsTest.java:71:31: 'typecast' is not followed by whitespace. [WhitespaceAfter]
[ant:checkstyle] [ERROR] .../UtilsTest.java:71:61: 'typecast' is not followed by whitespace. [WhitespaceAfter]
```

You can get this output by running:
```
gradle 
./gradlew clients:test
```

Reply:
Oh yes, seeing that now just pushed a style fix: deb060e.

As for the explicit import, I'm ok with what is here. We either have import declaration on line 35 or explicit import on line 62.